metadata-ingestion/source_docs/azure-ad.md

```diff
@@ -116,11 +121,11 @@ Note that a `.` is used to denote nested fields in the YAML configuration block.
 | `tenant_id`                            | string          | ✅        |                       | Directory ID. Found in your app registration on Azure AD Portal                                                                                                                                                                                            |                                     
-| `client_secret`                        | string          | ✅        |                       | Client secret. Found in your app registration on Azure AD Portal                                                                                                                                                                                           |                                  
-| `redirect`                             | string          | ✅        |                       | Redirect URI.  Found in your app registration on Azure AD Portal                                                                                                                                                                                           |                                  
+| `client_secret`                        | string          | ✅        |                       | Client secret. Found in your app registration on Azure AD Portal                                                                                                                                                                                           |                                                                    
 | `authority`                            | string          | ✅        |                       | The [authority](https://docs.microsoft.com/en-us/azure/active-directory/develop/msal-client-application-configuration) is a URL that indicates a directory that MSAL can request tokens from.                                                              |
 | `token_url`                            | string          | ✅        |                       | The token URL that acquires a token from Azure AD for authorizing requests.  This source will only work with v1.0 endpoint.                                                                                                                                |
-| `graph_url`                            | string          | ✅        |                       | [Microsoft Graph API endpoint](https://docs.microsoft.com/en-us/graph/use-the-api)                                                                                                                                                                         |
+| `redirect`                             | string          |          |                       | Redirect URI.  Found in your app registration on Azure AD Portal                                                                                                                                                                                           |
```

Comment:
minor: let's put in the comment what we default to if not provided.

Reply:
Done!